src/site/content/en/learn/css/sizing/index.md

```diff
@@ -0,0 +1,442 @@
+        Axis refers to writing modes.
+        In horizontal writing modes like English,
+        the inline axis is horizontal.
+        In vertical writing modes like some Japanese typefaces, the inline axis runs top to bottom.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vb">vb</a></td>
+        <td>1% of viewport's size in the root element's <a href="https://www.w3.org/TR/css-writing-modes-4/#block-axis">block axis</a>.
+        For the block axis, this would be the directionality of the language.
+        LTR languages like English would have a vertical block axis,
+        since English language readers parse the page from top to bottom.
+        A vertical writing mode has a horizontal block axis.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vmin">vmin</a></td>
+        <td>1% of the viewport's smaller dimension.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vmax">vmax</a></td>
+        <td>1% of the viewport's larger dimension.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+```css
+div {
+	width: 10vw;
+}
+
+p {
+	max-width: 60ch;
+}
+```
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'JjEYqXa'
+} %}
+</figure>
+
+In this example, the `div` will be 10% of the viewport's width because `1vw` is **1% of the viewport width**.
+The `p` element has a `max-width` of `60ch`
+which means it can't exceed the width of 60 "0" characters in the calculated font and size.
+
+{% Aside 'objective' %}
+
+By sizing text with relative units like `em` or `rem`,
+rather than an absolute unit, like `px`, the size of your text can respond to user preferences.
+This can include the system font size or parent element's font size, such as the `<body>`.
+The base size of the `em` is the element's parent and the base size of the `rem` is the base font size of the document.
+
+If you don't define a `font-size` on your `html` element,
+this user-preferred system font size will be honoured if you use relative lengths,
+such as `em` and `rem`.
+If you use `px` units for sizing text,
+this preference will be ignored.
+
+{% Video src="video/VbAJIREinuYvovrBzzvEyZOpw5w1/cUEl77VN4ZtAdElV4fd6.mp4" %}
+{% endAside %}
+
+## Miscellaneous units
+
+There are some other units which have been specified to deal with particular types of values.
+
+### Angle units
+
+In the [color module](/learn/css/color/),
+we looked at **angle units**,
+which are helpful for defining degree values,
+such as the hue in `hsl`.
+They are also useful for rotating elements within transform functions.
+
+```css
+div {
+	width: 150px;
+	height: 150px;
+	transform: rotate(60deg);
+}
+```
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'VwPvRbK'
+} %}
+</figure>
+
+Using the `deg` angle unit, you can rotate a `div` 90° on its center axis.
+
```

Comment:
```suggestion
```suggestion
Other angle units include `rad` (radians), `grad` (gradians), and `turn` units, which represent a part of an angle, where `1turn` = `360deg`, and `0.5turn` = `180deg`. These can come in handy when styling transforms and animations.
```
```